be/src/olap/rowset/segment_v2/column_reader.cpp

```diff
@@ -97,7 +97,7 @@ Status ColumnReader::read_page(RandomAccessFile* file, const PagePointer& pp,
     auto cache = StoragePageCache::instance();
     PageCacheHandle cache_handle;
     StoragePageCache::CacheKey cache_key(file->file_name(), pp.offset);
-    if (cache->lookup(cache_key, &cache_handle)) {
+    if (!config::disable_storage_page_cache && cache->lookup(cache_key, &cache_handle)) {
```

Comment:
It is better to use an argument for this function.
Besides the configuration, some operation don't need cache either, such as compaction.

So the good way is that let the caller judge whether the cache is used.

Reply:
OK, I agree with you.